renovate.json

```diff
@@ -5,7 +5,7 @@
   ],
   "packageRules": [
     {
-      "matchFields": [
+      "matchFiles": [
```

Comment:
This was intended to be https://docs.renovatebot.com/configuration-options/#matchfiles